__tests__/endpoints/getCostumeById.test.js

```diff
@@ -0,0 +1,47 @@
+const {
+    getPool,
+    createTables, 
+    createCostume, 
+} = require('../../db/');
+
+const {
+    getBallroomGown,
+    getButtlessChaps,
+    getBonnet
+} = require('../../db/utils');
+
+const {server} = require('../../app')
+const supertest = require('supertest');
+const request = supertest(server)
+
+
+let pool = getPool();
+
+describe('GET api/costumes/:id', () => {
+    afterAll(async () => {
+        await pool.end()
+    })
+    
+    it('returns costume by ID', async () => {
+        await createTables(pool);
+        await createCostume(pool, getBallroomGown());
+        await createCostume(pool, getButtlessChaps());
+        await createCostume(pool, getBonnet());
```

Comment:
You can move these into a `beforeEach` since they are reused for both tests

Reply:
So, I was getting an open handle error from Jest when I did that. Apparently other have had this issue with supertest. Even now without beforeEach it is doing a weird thing where Jest says it was not able to exit in time and that there may be open handles, but when I run it with `--detectOpenHandles` it works fine. An issue like this has been documented on Jest's GitHub: https://github.com/jestjs/jest/issues/6937

Reply:
Here are some screenshots:

With `beforeEach` call:
![Screen Shot 2023-09-19 at 16 32 59](https://github.com/dyazdani/costume-shop-db/assets/99094815/53950ef7-6ad1-4af3-a654-edc866c4e9f0)


With `beforeEach` call and run with `--detectOpenHandles`:
![Screen Shot 2023-09-19 at 16 34 50](https://github.com/dyazdani/costume-shop-db/assets/99094815/c1076ba6-e44e-4f21-8c03-461121dc0a93)


Without `beforeEach` call:
![Screen Shot 2023-09-19 at 16 32 59](https://github.com/dyazdani/costume-shop-db/assets/99094815/5f610006-175b-4bf5-b937-667fac6bc86c)


Without `beforeEach` call and run with `--detectOpenHandles`:
![Screen Shot 2023-09-19 at 16 37 45](https://github.com/dyazdani/costume-shop-db/assets/99094815/061b1ca5-a4c9-4ad4-a7ae-a43c220c8061)

Reply:
Interesting. Maybe an issue with supertest then? If the tests are passing as is, and everything works in postman, I think you're fine for now.

I'm not sure if there's a way to cleanup open supertest requests

Reply:
I can look in the supertest docs